src/components/AdditionalFeature.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import { connect } from 'tls';
 
 const AdditionalFeature = props => {
   return (
     <li>
       {/* Add an onClick that will let you add a feature to your car */}
-      <button className="button">Add</button>
+      <button onClick={(e) => props.buyItem(e, props.feature)} className="button">Add</button>
```

Comment:
I under stand the impulse to write your entire funtion into the onClick, but in order to gain more control of your code, I would suggest breaking them out and hoisting your function above the return in the function, It makes debugging easier and your code more read able